### ISSUE TITLE
WebSockets package updated which breaks certstream

### ISSUE THREAD

**Issue by HugooB** [CaliDog/certstream-python#42]
Hi!

It seems that https://github.com/websocket-client/websocket-client has changed something, which causes certstream to fail. Could somebody fix this issue? 

```python
error from callback <bound method CertStreamClient._on_open of <certstream.core.CertStreamClient object at 0x1185f6e20>>: _on_open() takes 1 positional argument but 2 were given
```

Similar to #39 ?

**Quick fix**
Pin requirements to `websocket-client==0.57.0`

**Comment by srknzl** [comment]
Using certstream==1.10 fixes for me

**Comment by Fitblip** [comment]
Hi folks, this should be fixed with #42

**Comment by retpoline** [comment]
Awesome, thanks @Fitblip!
